Tests/DownloadTests.swift

```diff
@@ -226,13 +204,13 @@ class DownloadResponseTestCase: BaseTestCase {
         let fileURL = randomCachesFileURL
         let urlString = "https://httpbin.org/get"
         let headers = ["Authorization": "123456"]
-        let destination: DownloadRequest.DownloadFileDestination = { _, _ in fileURL }
+        let destination: DownloadRequest.DownloadFileDestination = { _, _ in (fileURL, []) }
 
         let expectation = self.expectation(description: "Download request should download data to file: \(fileURL)")
         var response: DefaultDownloadResponse?
 
         // When
-        Alamofire.download(urlString, to: destination, withMethod: .get, headers: headers)
+        Alamofire.download(urlString, method: .get, headers: headers, to: destination)
```

Comment:
Will remove. At first I kept some around just to switch things up a bit to make sure everything was working correctly.